src/10_functions.py

```diff
@@ -10,3 +10,10 @@
 
 # YOUR CODE HERE
 
+
+def is_even(num):
```

Comment:
Function works great; the instructions do ask for "Even!" and "Odd" printouts, not just True and False. I know that might seem nitpicky, but later on the directions can get very particular, so always good to work on following them to the letter.